WThermostat/src/WBecaDevice.h

```diff
@@ -41,15 +42,20 @@ const byte STORED_FLAG_BECA = 0x36;
 const char SCHEDULES_PERIODS[] = "123456";
 const char SCHEDULES_DAYS[] = "wau";
 
-const float MODEL_TEMPERATURE_FACTOR[]                  = { 2.0f, 2.0f, 10.0f, 10.0f };
-const byte  MODEL_MCU_BYTE_TEMPERATURE_TARGET[]         = {0x02, 0x02, 0x02, 0x02 };
-const byte  MODEL_MCU_BYTE_TEMPERATURE_ACTUAL[]         = {0x03, 0x03, 0x08, 0x03 };
-const byte  MODEL_MCU_BYTE_TEMPERATURE_FLOOR[]          = {0x66, 0x00, 0x05, 0x66 };
-const byte  MODEL_MCU_BYTE_SYSTEM_MODE[]  			        = {0x00, 0x66, 0x00, 0x00 };
-const byte  MODEL_MCU_BYTE_SCHEDULES_MODE[]			        = {0x04, 0x04, 0x03, 0x04 };
-const byte  MODEL_MCU_BYTE_FAN_MODE[]  					        = {0x00, 0x67, 0x00, 0x00 };
-const byte  MODEL_MCU_BYTE_ECO_MODE[]  					        = {0x05, 0x05, 0x00, 0x00 };
-const byte  MODEL_MCU_BYTE_SCHEDULES[]                  = {0x65, 0x68, 0x00, 0x00 };
+const float MODEL_TEMPERATURE_FACTOR[]                  = {2.0f, 2.0f, 10.0f, 10.0f, 1.0f };
+const byte  MODEL_MCU_BYTE_TEMPERATURE_TARGET[]         = {0x02, 0x02, 0x02, 0x02, 0x10 };
+const byte  MODEL_MCU_BYTE_TEMPERATURE_ACTUAL[]         = {0x03, 0x03, 0x08, 0x03, 0x18 };
+const byte  MODEL_MCU_BYTE_TEMPERATURE_FLOOR[]          = {0x66, 0x00, 0x05, 0x66, 0x2d };
+const byte  MODEL_MCU_BYTE_SYSTEM_MODE[]  			    = {0x00, 0x66, 0x00, 0x00, 0x24 };
+const byte  MODEL_MCU_BYTE_SCHEDULES_MODE[]			    = {0x04, 0x04, 0x03, 0x04, 0x02 };
+const byte  MODEL_MCU_BYTE_FAN_MODE[]  					= {0x00, 0x67, 0x00, 0x00, 0x00 };
+const byte  MODEL_MCU_BYTE_ECO_MODE[]  					= {0x05, 0x05, 0x00, 0x00, 0x00 };
+const byte  MODEL_MCU_BYTE_SCHEDULES[]                  = {0x65, 0x68, 0x00, 0x00, 0x26 };
+const byte  MODEL_MCU_BYTE_LOCKED[]                     = {0x06, 0x06, 0x06, 0x06, 0x28 };
+const int   MODEL_SCHEDULING_DAYS[]                     = {18,   18,   18,   18,   8 };
```

Comment:
The new Thermostat provides only 8 periods, not 18.
The scheduling command for the new thermostat has per period HH MM TT, the old one MM HH TT - this will keep it flexible.